.azure/applications/bff-frontend-poc/main.bicep

```diff
@@ -0,0 +1,33 @@
+targetScope = 'resourceGroup'
+
+@minLength(3)
+param imageTag string
+@minLength(3)
+param environment string
+@minLength(3)
+param location string
+
+@minLength(3)
+@secure()
+param containerAppEnvironmentName string
+
+var namePrefix = 'dp-fe-${environment}'
+var baseImageUrl = 'ghcr.io/digdir/dialogporten-frontend-'
+var containerAppName = '${namePrefix}-bff-frontend-poc'
+
+resource containerAppEnvironment 'Microsoft.App/managedEnvironments@2023-05-01' existing = {
+  name: containerAppEnvironmentName
+}
+
+module containerApp '../../modules/containerApp/main.bicep' = {
+  name: containerAppName
+  params: {
+    name: containerAppName
+    location: location
+    image: '${baseImageUrl}bff-frontend-poc:${imageTag}'
```

Comment:
Why doesn't this use the variable `containerAppName`? is `containerAppName` something else than image name?

Reply:
Yes good point